dfaligner/model.py

```diff
@@ -6,7 +6,7 @@
 import torch
 import torch.nn as nn
 from everyvoice.model.aligner.config import AlignerConfig
-from everyvoice.text import TextProcessor
+from everyvoice.text.text_processor import TextProcessor
```

Comment:
Rename `text_processor` for simply `processor`.  It's already in the `text` module and the class already has `Text` in it's name.

Reply:
As mentioned in the main repo, I think I'll keep it this way